cmd/user-agent/src/pages/IssueCredential.vue

```diff
@@ -0,0 +1,382 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
+    <div class="content">
+        <md-dialog :md-active.sync="dialog">
+            <md-dialog-title>{{ dialogTitle }}</md-dialog-title>
+            <div class="content">
+                <div class="md-layout">
+                    <div class="md-layout-item">
+                        <div v-if="dialogContent === 'acceptRequestForm'">
+                            <md-field>
+                                <label>JSON IssueCredential</label>
```

Comment:
why is it constrained to JSON?

Reply:
It is a full message. It might contain a lot of fields. How it should look on UI I do not know.
The current implementation assumes that the user will provide the whole message (whatever he want).
e.g
```
// TODO: Need to add ~please-ack decorator [Issue #1299]
type IssueCredential struct {
	Type string `json:"@type,omitempty"`
	// Comment is an optional field that provides human readable information about this Credential Offer,
	// so the offer can be evaluated by human judgment.
	// TODO: Should follow DIDComm conventions for l10n. [Issue #1300]
	Comment string `json:"comment,omitempty"`
	// CredentialsAttach is a slice of attachments containing the issued credentials.
	CredentialsAttach []decorator.Attachment `json:"credentials~attach,omitempty"`
}
```
NOTE: The `Type` is always set by the framework. Other fields are not.

I can create a more friendly UI. But I am not sure that we really need it. And how to render it (I mean how it should look like). 
Let me know what kind of improvement would you like to see.

Reply:
I got it. I can use `select` to choose the credential from the store. And I will add an additional field for comment. @llorllale  Make sense, right?

Reply:
@soluchok credentials are not normally laying around in your store waiting for you to issue them when someone asks. They're usually created and issued on demand. Of course, you may save them after creation.